.github/workflows/manual-build.yml

```diff
@@ -0,0 +1,41 @@
+name: "Build Manual"
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
This should only trigger, when there are pushes to doc/**

Do we want to also add the scheduled one we have for the other builds?